openedx/adg/lms/applications/test/factories.py

```diff
@@ -15,3 +17,15 @@ class Meta:
 
     title = factory.Faker('word')
     description = factory.Faker('sentence')
+
+
+class ExtendedUserProfileFactory(factory.django.DjangoModelFactory):
+    """
+    Factory for ExtendedUserProfile model
+    """
+
+    class Meta:
+        model = ExtendedUserProfile
+
+    user = factory.SubFactory(UserFactory)
+    saudi_national = False
```

Comment:
Please rebase your branch from latest master. We already have a factory for this in master. registration_extension/tests/factories.py